manageiq-operator/README.md

```diff
@@ -72,13 +162,15 @@ metadata:
   name: miq
 spec:
   applicationDomain: "miqproject.apps-crc.testing"
+  httpdAuthenticationType: openid-connect
+  oidcProviderURL: https://<your keycloak FQDN>/auth/realms/<your Keycloak client name>/.well-known/openid-configuration
```

Comment:
`<your Keycloak client name>` should be `<your keycloak Realm>`

Reply:
Ah yes! Of course. Good catch!